### PR TITLE
Use conftest from ragger

### PR DESCRIPTION
**Pull request by fbeutin-ledger**

### CODE REVIEW

File: .github/workflows/reusable_swap_functional_tests.yml [hunk 1]
```diff
@@ -77,62 +72,40 @@ jobs:
...
-          make clean ; make COIN=${{ matrix.coin.name }} BOLOS_SDK=${{ matrix.sdk.path }} DEBUG=1
-          cp bin/app.elf bin/${{ matrix.coin.name }}_${{ matrix.sdk.name }}.elf
-
-      - name: Upload ${{ matrix.coin.name }} binaries
-        uses: actions/upload-artifact@v3
-        with:
-          name: coins_binaries
-          path: ./bin/${{ matrix.coin.name }}_${{ matrix.sdk.name }}.elf
-          if-no-files-found: error
+    uses: LedgerHQ/ledger-app-workflows/.github/workflows/reusable_build.yml@fbe/more_options
```
Comment (xchapron-ledger) [inline .github/workflows/reusable_swap_functional_tests.yml:75]: @ should be changed before merging :)

File: .gitignore [hunk 2]
```diff
@@ -29,15 +29,12 @@ _ReSharper*/
 [Tt]est[Rr]esult*
 .vs/
 #Nuget packages folder
-test/elfs
 packages/
-bin
-debug
```
Comment (xchapron-ledger) [inline .gitignore:35]: this should be keep

Comment (fbeutin-ledger) [reply .gitignore:35]: Done

File: .github/workflows/reusable_swap_functional_tests.yml (hunk 1 above)
Comment (fbeutin-ledger) [reply .github/workflows/reusable_swap_functional_tests.yml:75]: I'll keep open to remind me

**Pull request merged**